numpy/polynomial/tests/test_legendre.py

```diff
@@ -100,6 +100,18 @@ def test_legdiv(self):
                 res = leg.legadd(leg.legmul(quo, ci), rem)
                 assert_equal(trim(res), trim(tgt), err_msg=msg)
 
+    def test_legpow(self):
+        for i in range(5):
+            for j in range(5):
+                msg = "At i=%d, j=%d" % (i, j)
+                c = np.array(range(i + 1))
+                power = j + 1
+                tgt = c
+                for i in range(2, power + 1):
```

Comment:
This overwrites the `i` variable, which could end up confusing someone editing this test. Why not `range(1, power)` also?